cs-office-365/src/main/java/io/cloudslang/content/office365/actions/utils/GetAuthorizationToken.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * (c) Copyright 2019 Micro Focus, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.cloudslang.content.office365.actions.utils;
+
+import com.hp.oo.sdk.content.annotations.Action;
+import com.hp.oo.sdk.content.annotations.Output;
+import com.hp.oo.sdk.content.annotations.Param;
+import com.hp.oo.sdk.content.annotations.Response;
+import com.microsoft.aad.adal4j.AuthenticationResult;
+import io.cloudslang.content.constants.ReturnCodes;
+import io.cloudslang.content.office365.entities.AuthorizationTokenInputs;
+import io.cloudslang.content.office365.services.AuthorizationTokenImpl;
+import io.cloudslang.content.utils.NumberUtilities;
+import io.cloudslang.content.utils.StringUtilities;
+
+import java.util.List;
+import java.util.Map;
+
+import static com.hp.oo.sdk.content.plugin.ActionMetadata.MatchType.COMPARE_EQUAL;
+import static com.hp.oo.sdk.content.plugin.ActionMetadata.ResponseType.ERROR;
+import static com.hp.oo.sdk.content.plugin.ActionMetadata.ResponseType.RESOLVED;
+import static io.cloudslang.content.constants.OutputNames.*;
+import static io.cloudslang.content.constants.ResponseNames.FAILURE;
+import static io.cloudslang.content.constants.ResponseNames.SUCCESS;
+import static io.cloudslang.content.office365.utils.Constants.*;
+import static io.cloudslang.content.office365.utils.Descriptions.Common.*;
+import static io.cloudslang.content.office365.utils.Descriptions.GetAuthorizationToken.*;
+import static io.cloudslang.content.office365.utils.Inputs.AuthorizationInputs.*;
+import static io.cloudslang.content.office365.utils.Inputs.CommonInputs.*;
+import static io.cloudslang.content.office365.utils.InputsValidation.verifyAuthorizationInputs;
+import static io.cloudslang.content.office365.utils.Outputs.AuthorizationOutputs.AUTH_TOKEN_TYPE;
+import static io.cloudslang.content.utils.OutputUtilities.getFailureResultsMap;
+import static io.cloudslang.content.utils.OutputUtilities.getSuccessResultsMap;
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.apache.commons.lang3.StringUtils.defaultIfEmpty;
+
+public class GetAuthorizationToken {
+
+    /**
+     * @param loginType      Login method according to application type
+     *                       Valid values: 'API', 'Native'
+     *                       Default: 'API'
```

Comment:
Please remove description from here

Reply:
Personally, I would like to keep this description for other developers to see it when opening the action. 👍